src/main/scala/com/typesafe/zinc/Compiler.scala

```diff
@@ -218,4 +226,46 @@ class Compiler(scalac: AnalyzingCompiler, javac: JavaCompiler) {
   }
 
   override def toString = "Compiler(Scala %s)" format scalac.scalaInstance.actualVersion
+
+  private[this] def makeInputs(inputs: Inputs, cwd: Option[File], compileReporter: Reporter, compileProgress: Option[CompileProgress], log: Logger): xsbti.compile.Inputs[Analysis, AnalyzingCompiler] = {
+    def o2m[T](opt: Option[T]): Maybe[T] = opt map (Maybe.just(_)) getOrElse Maybe.nothing()
+
+    import inputs._
+    new xsbti.compile.Inputs[Analysis, AnalyzingCompiler] {
+      override val compilers = new Compilers[AnalyzingCompiler] {
+        override val scalac = self.scalac
+        override val javac = new xsbti.compile.JavaCompiler {
+          override def compileWithReporter(sources: Array[File], classpath: Array[File], output: Output, options: Array[String], reporter: Reporter, log: Logger): Unit = {
+            // FIXME: Classpath???
```

Comment:
Should the class path be specified with something such as
```scala
val optionsWithClasspath = options :+ ("-cp" + (classpath mkString ":"))
```

That's what the javadoc suggest for `run`?